src/util/chromium-preferences.js

```diff
@@ -0,0 +1,33 @@
+import { UsageError } from '../errors.js';
+
+export function coerceCLICustomChromiumPreference(cliPrefs) {
+  const customPrefs = {};
+
+  for (const pref of cliPrefs) {
+    const prefsAry = pref.split('=');
+
+    if (prefsAry.length < 2) {
+      throw new UsageError(
+        `Incomplete custom preference: "${pref}". ` +
+          'Syntax expected: "prefname=prefvalue".',
+      );
+    }
+
+    const key = prefsAry[0];
+    let value = prefsAry.slice(1).join('=');
+
+    if (/[^\w_.]/.test(key)) {
```

Comment:
Is there any point in this check or can it be dropped?

`\w` means `[A-Za-z0-9_]`, so what your code here does is rejecting anything other than alphanumeric characters and period.

It is stricter than the actual values permitted by Chromium's Preferences keys, which is quite permissive actually. For example it can contain URLs (including `:` characters which are rejected by your current code), e.g. as seen at:
- example of file: https://source.chromium.org/chromium/chromium/src/+/main:chrome/test/data/diagnostics/user/Default/Preferences;l=58;drc=f0f6afc99205acab3f069d915cdad756a52d8809
- example in code comment: https://source.chromium.org/chromium/chromium/src/+/main:net/http/http_server_properties_manager.cc;l=245-256;drc=4787fce6c51383f5631643ac3d14cc512d656de6